### PR TITLE
Update basemap doc

### PR DESCRIPTION
**Pull request by nekhtiari**
Pull Request
============

> Tracked in JIRA by [IN-XXX](https://geoinformationstore.atlassian.net/browse/IN-XXX)
> Paired with @a, @b

### Scope of the PR

This PR added documentation for oneatlas-basemap block. Please note that the block url needs to be updated.

### Checklist

- [x] Checked spelling and grammar
- [ ] Provided code samples where relevant
- [x] Checked the output of `make html` to ensure new content displays correctly

### CODE REVIEW

**Review by teoEnache — APPROVED** [review]
LGTM

**Review by rodrigoalmeida94 — APPROVED** [review]
LGTM

File: source/up42-blocks/data/oneatlas-basemap.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
+.. meta::
+   :description: UP42 data blocks: Oneatlas Basemap data block description
+   :keywords: "OneAtlas", "global", "satellite", "high resolution", "pleiades", "spot", "map"
+
+.. _oneatlas-basemap-block:
+
+Airbus Basemap (Tile)
+===================================
+
+For more information, please read the ...
```
Comment (markusuwe) [inline source/up42-blocks/data/oneatlas-basemap.rst:10]: Dangling sentence, copy/paste maybe?

File: source/up42-blocks/data/oneatlas-basemap.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+Supported parameters
+--------------------
+
+OneAtlas Basemap is a worldwide off-the-shelf imagery layer made from Airbus highest-grade satellite imagery.
+These images are sharp and will be updated constantly. Therefore, they provide reliable context. One Atlas basemap
+offers a complete coverage for the whole world with high resolution (1.5m over the world, 50cm over the most populated urban areas)
+and precise geolocation. New images will be added every day, and the whole archive will be refreshed in a yearly basis.
+Images have a marginal cloud cover.
+
+* ``bbox``: The bounding box to use as an AOI. Will return all geometries that intersect with this box. Only ``box`` is supported.
```
Comment (markusuwe) [inline source/up42-blocks/data/oneatlas-basemap.rst:23]:
```suggestion
* ``bbox``: The bounding box to use as an AOI. Will return all geometries that intersect with this box.
```

File: source/up42-blocks/data/oneatlas-basemap.rst [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+* ``intersects``: A GeoJSON geometry to use as an AOI. Will return all scenes that intersect with this geometry. Use only ``bbox``
+  **or** ``intersects``.
+* ``zoom_level``: An integer defining the webmercator zoom level of this request, defaults to 18.
+
+.. note::
+
+  OneAtlas orders the imagery layers first by constellation (Pléiades then SPOT) and for each constellation by acquisition date.
+  Based on your chosen AOI, you will get tiles from different layers to cover your area of interest. Please note that
+  the final metadata file only includes the information for the first map layer.
```
Comment (markusuwe) [inline source/up42-blocks/data/oneatlas-basemap.rst:32]: What does first map layer mean in this context?

**Review by markusuwe — COMMENTED** [review]
LGTM

File: source/up42-blocks/data/oneatlas-basemap.rst (hunk 1 above)
Comment (nekhtiari) [reply source/up42-blocks/data/oneatlas-basemap.rst:10]: Yes, I added in PR desc that I need to update this!

File: source/up42-blocks/data/oneatlas-basemap.rst (hunk 3 above)
Comment (nekhtiari) [reply source/up42-blocks/data/oneatlas-basemap.rst:32]: The newest(timewise?) map layer I meant

File: source/up42-blocks/data/oneatlas-basemap.rst (hunk 2 above)
Comment (nekhtiari) [reply source/up42-blocks/data/oneatlas-basemap.rst:23]: Removed!

File: source/up42-blocks/data/oneatlas-basemap.rst (hunk 1 above)
Comment (markusuwe) [reply source/up42-blocks/data/oneatlas-basemap.rst:10]: Apologies, overlooked that.

File: source/up42-blocks/data/oneatlas-basemap.rst (hunk 3 above)
Comment (markusuwe) [reply source/up42-blocks/data/oneatlas-basemap.rst:32]: Maybe better to clarify.

**Review by thaisbendixen42 — APPROVED** [review]

**Pull request merged**